### PR TITLE
Add long explanation for E0547

### PR DESCRIPTION
**Pull request by jesusprubio**
Helps with #61137

### CODE REVIEW

**Review by GuillaumeGomez — APPROVED** [review]

**Comment by GuillaumeGomez** [comment]
Thanks!

@bors: r+ rollup squash

**Comment by GuillaumeGomez** [comment]
Generally we use rebase instead of merge to fix conflicts. However, I don't know if it's a requirement or not...

File: compiler/rustc_error_codes/src/error_codes/E0547.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+The `issue` value is missing in a stability attribute.
```
Comment (pickfire) [inline compiler/rustc_error_codes/src/error_codes/E0547.md:1]: Would this be clearer if it is flipped? I wonder if the term is called "value"?
```suggestion
A stability attribute is missing `issue`.
```

Comment (jesusprubio) [reply compiler/rustc_error_codes/src/error_codes/E0547.md:1]: Honestly, I can't see any benefit. Anyway, it's not a big deal for me :).

@GuillaumeGomez , what do you think?

**Comment by jesusprubio** [comment]
> Generally we use rebase instead of merge to fix conflicts. However, I don't know if it's a requirement or not...

My fault :), I'm starting a new PR and closing this one.

File: compiler/rustc_error_codes/src/error_codes/E0547.md (hunk 1 above)
Comment (GuillaumeGomez) [reply compiler/rustc_error_codes/src/error_codes/E0547.md:1]: I find the suggested one more difficult to understand.

**Comment by jesusprubio** [comment]
Closing in favor of: https://github.com/rust-lang/rust/pull/81925

**Pull request closed without merging**